hack/tools/go.mod

```diff
@@ -12,10 +12,9 @@ require (
 	github.com/onsi/ginkgo v1.16.1
 	github.com/onsi/gomega v1.11.0 // indirect
 	k8s.io/apiextensions-apiserver v0.21.0 // indirect
-	k8s.io/code-generator v0.21.0
-	k8s.io/utils v0.0.0-20210305010621-2afb4311ab10 // indirect
+	k8s.io/code-generator v0.21.1
 	sigs.k8s.io/cluster-api/hack/tools v0.0.0-20210402170612-77093e2350cd
 	sigs.k8s.io/controller-tools v0.5.0
-	sigs.k8s.io/kind v0.10.0
+	sigs.k8s.io/kind v0.11.0
```

Comment:
We can use v0.11.1, But It is ok when updating to v0.4.0.
https://github.com/kubernetes-sigs/kind/releases/tag/v0.11.1

Reply:
fixed